apps/schedclock/metadata.json

```
@@ -0,0 +1,23 @@
+{ "id": "schedclock",
+  "name": "Schedule Clock Faces",
+  "shortName":"Sched Clock",
+  "version":"0.01",
+  "author": "kidneyhex",
+  "description": "Change clock faces on a schedule.",
+  "icon": "app.png",
+  "screenshots": [
+    {"url":"screenshot1.png"},
+    {"url":"screenshot2.png"}
+  ],
+  "tags": "tool",
+  "supports" : ["BANGLEJS2"],
+  "readme": "README.md",
+  "dependencies": {"scheduler":"type"},
+  "storage": [
+    {"name":"schedclock.app.js","url":"app.js"},
+    {"name":"schedclock.settings.js","url":"settings.js"},
+    {"name":"schedclock.lib.js","url":"lib.js"},
```

```suggestion
    {"name":"schedclock","url":"lib.js"},
```

I'm more used to this suggested pattern for loading libraries to the watch. But I see there are also apps that do it like you have it here (`ha`, `timerclk`, `tineheads`, `xxlmessage`).

@gfwilliams do we have a preference for how to name libraries on the watch?

Note: If you want to change there would need to be accommodating changes at other places in the app as well.

---

Side question: Is there a recommended way to migrate old settings?

I just tested adding a `{"url": "migrate_v01_v02.js", "evaluate":true}` to the `metadata.json`, and as long as I put a `console.log("OK\r");` at the end of the migrate.js, it worked. But I can't find anything in docs/discussions about it.